DCOS_OSS-3581: prettier config is invalid

DanielMSchmidt opened the pull request:
Currently, prettier is not executed as the config provided expected either a boolean or an array of the length two. 

Closes DCOS_OSS-3581

Commit:
fix: prettier config is invalid

Currently, prettier is not executed as the config provided expected either a boolean or an array of the length two.

Closes DCOS_OSS-3581

juliangieseke reviewed (changes requested):
hm, I tried to link this package on your branch and run tslint, but it didnt complain about style things 🤔 after adding to `tslint.json` in dcos-ui it worked…

Commit:
fix: half of the tslint config was not included in the rules

BREAKING CHANGE: This enables a lot of rules, which means projects using
        this library will most probably need to adapt

juliangieseke reviewed (approved):
now it explodes 💥 (which is good, because that means it works :D)

Poltergeist reviewed (approved):
☕️ 🇹🇬
